tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -366,7 +372,11 @@ function render<T extends boolean = true>(
       config,
     });
 
-    renderResult.rerender(<RouterProvider router={newRouter} />);
+    renderResult.rerender(
+      <RouterProvider router={newRouter} future={{v7_startTransition: true}} />
+    );
+    // Force the router to update children
+    rtl.act(() => newRouter.revalidate());
```

Comment:
rerenders were broken without this piece. If anyone has other ideas i'm not sure why we needed this.